core/modules/filters/getstoryviewsingletiddlermode.js

```diff
@@ -0,0 +1,35 @@
+/*\
+title: $:/core/modules/filters/getstoryviewsingletiddlermode.js
+type: application/javascript
+module-type: filteroperator
+
+Filter operator for retrieving the single tiddler mode status of a storyview.
+
+\*/
+(function(){
+
+/*jslint node: true, browser: true */
+/*global $tw: false */
+"use strict";
+
+/*
+Export our filter functions
+*/
+
+exports.getstoryviewsingletiddlermode = function(source,operator,options) {
```

Comment:
wouldn't it be better to have a function "getstoryviewmode", that returns the mode name? ... 

IMO we would need a new function for every other existing and future mode too!

Reply:
Hi @pmario good point, I'll fix.